examples/key_release/payload.py

```diff
@@ -42,30 +21,18 @@ def do_POST(self):
         self.wfile.write(response.content)
 
     def do_GET(self):
-        if self.path in ENDPOINTS:
-            try:
-                result = ENDPOINTS[self.path]()
-                self.send_response(200)
-            except Exception as e:
-                result = str(e)
-                self.send_response(400)
-
-            self.send_header("Content-type", "text/plain")
-            self.end_headers()
-            self.wfile.write(result.encode("utf-8"))
-        else:
-            response = requests.get(
-                url=f"http://localhost:8080{self.path}",
-                headers=dict(self.headers),
-                data=self.rfile.read(int(self.headers.get("Content-Length", 0))),
-            )
-            self.send_response(response.status_code)
-            for header, value in response.headers.items():
-                if header != "Transfer-Encoding":
-                    self.send_header(header, value)
-            self.end_headers()
+        response = requests.get(
+            url=f"http://localhost:8080{self.path}",
+            headers=dict(self.headers),
+            data=self.rfile.read(int(self.headers.get("Content-Length", 0))),
```

Comment:
since it's a get request, do we need to send a body?

Reply:
@DomAyre originally created this file, so I would defer to him as to whether there's a reason he did the Key Release test this way.

Reply:
I don't remember that well, it could easily be a copy paste typo from the POST equivalent. These just pass requests onto the sidecar so (hopefully) assuming the sidecars don't use a body in any GET requests we can safely remove it